wsclient/src/main/java/org/hpccsystems/ws/client/platform/DFUDataColumnInfo.java

```diff
@@ -3,15 +3,19 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import org.hpccsystems.ws.client.gen.wsdfu.v1_36.DFUDataColumn;
-
 // This class wraps the generated soap ECL Workunit, providing comparable and to-string methods for end-users.
-public class DFUDataColumnInfo extends DFUDataColumn
+public class DFUDataColumnInfo
 {
-    /**
-     * 
-     */
-    private static final long       serialVersionUID = 1L;
```

Comment:
I can't tell, but if we serialize this class, we might want to keep serialVersionUID, and update it

Reply:
It's not needed as a top level class now that it's no longer extending DFUDataColumn, it has its own serial uid generated